components/ExerciseReportCard/exerciseReportCard.module.scss

```diff
@@ -0,0 +1,30 @@
+@use '../../scss/colors.module.scss';
+
+.container {
+  max-width: 400px;
+  background-color: hsl(0, 0%, 96%);
+  padding: 20px;
+  border: 1px solid hsl(0, 0%, 84%);
+  border-radius: 10px;
+}
+
+.container__reportBtn {
```

Comment:
We do have styles for the buttons which kind of overlap, would it be possible to use that and extend it instead?
https://github.com/garageScript/c0d3-app/blob/master/scss/button.module.scss

Reply:
Sure. Good idea.